conf/cms/de/home.yaml

```diff
@@ -1,68 +1,58 @@
-bannerTopLeft:
+banners[0]:
   title: Sommer Essentials
   subtitle: Die wichtigsten Stücke des Sommer Trends 2016
-  linkText: Jetzt bestellen
-  linkHref: men
-  imageSmall: /assets/img/cms/leather1_small-4to3.jpg
-  imageMedium: /assets/img/cms/leather1_medium-4to3.jpg
-  imageLarge: /assets/img/cms/leather1_large-4to3.jpg
+  linkLabel: Jetzt bestellen
+  link: /de/men
+  backgroundImage: /assets/img/cms/leather1-4to3.jpg
```

Comment:
Ah because before we used 3 different entries for images: small, medium, large.
But that disabled the possibility to use media capabilities of CMS, such as modifying size via query parameters (while keeping only one image entry on your content).
So we'll introduce passing options via handlebars helper and then each CMS implementation will define how these parameters are handled. That avoids entry duplication (imageSmall, imageMedium, imageLarge).
Of course once we have this new approach, we'll have to recover the images in different sizes, but for now I chose to removed them completely from the Theme to avoid confusion.